README.md

```diff
@@ -69,10 +69,19 @@ After the main image has been configured the process of exporting the image, cre
 
 Once you are happy with the main image the process is as follows:
 
+Repo Location:
+The repo should be installed to C:\
+
+If the repo is already there, cd into it and perform a git pull.
+
+If the repo is not already there, cd into C:\ and perform `git clone https://github.com/IamSamD/vdi-wsl-setup.git`
+
+Once the repo is there and up to date:
+
 - Set up your config file with the desired values
 - Run the scripts in the following order
 
-Export the main distro
+Check if there is already an export of the main distro, this will be called something like Ubuntu.tar.  If there is not already a main distro available:
```

Comment:
The main distro export is scripted to be called `main-distro.tar` which you can see on this line of the export script:

https://github.com/IamSamD/vdi-wsl-setup/blob/27a407adb222789a32845370fbc1e8eccdbe19b8/Invoke-ExportWslDistros.ps1#L41

Advise correcting that for clarity.